phpBB/adm/style/acp_ext_catalog.html

```diff
@@ -0,0 +1,130 @@
+{% include('overall_header.html') %}
+
+<a id="maincontent"></a>
+
+<h1>{{ lang( 'EXTENSIONS_CATALOG') }}</h1>
+
+<p>{{ lang( 'EXTENSIONS_CATALOG_EXPLAIN') }}</p>
+
+<fieldset class="quick quick-left">
+	<span class="small"><a href="https://www.phpbb.com/go/customise/extensions/3.1" target="_blank">{{ lang('BROWSE_EXTENSIONS_DATABASE') }}</a> &bull; <a href="javascript:phpbb.toggleDisplay('catalog_settings');">{{ lang('SETTINGS') }}</a></span>
```

Comment:
I think we should also work a bit on stuff like inline CSS and inline JS though that can and should IMHO happen after this is merged.